frontend/public/src/lib/notificationsApi.js

```diff
@@ -55,6 +56,10 @@ export function parseStoredUserMessage(
     msgText =
         "We're sorry, your country is currently blocked from enrolling in this course."
     break
+  case USER_MSG_TYPE_ENROLL_DUPLICATED:
+    alertType = ALERT_TYPE_DANGER
+    msgText = "You have already enrolled and paid for this course."
```

Comment:
Please include an email address in the message. 

"You have already enrolled and paid for this course. If this is unexpected, please contact customer support at {support_email}"

Reply:
@pdpinch I've updated the message